app/controllers/staff/training_sessions_controller.rb

```diff
@@ -156,7 +171,15 @@ def certify_trainees
     respond_to do |format|
       format.html do
         redirect_to staff_dashboard_index_path,
-                    notice: "Training Session Completed Successfully"
+                    notice:
+                      "Training Session Completed Successfully" +
+                        (
```

Comment:
would prefer `duplicates > 0 ? " (#{duplicates} duplicates skipped)" : ""`

Reply:
So uh, prettier actually converts the ternary into that.